src/main/java/core/basesyntax/dao/FileDaoCsvImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.dao;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileDaoCsvImpl implements FileDaoCsv {
+    @Override
+    public List<String> getData(String fileName) {
+        List<String> operations = new ArrayList<>();
```

Comment:
```suggestion
        List<String> data = new ArrayList<>();
```
operations are `b, s, p, r`